src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -143,10 +147,12 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
 
         const datasets = _.flattenDeep(cancerTypeStudiesArray.map((cancerStudySet, i) => (
             cancerStudySet.studies.sort((a: CancerStudy, b:CancerStudy) => b.allSampleCount - a.allSampleCount).map((cancerStudy: CancerStudy, j: number) => {
+                let int = 3;
                 const {name, studyId, allSampleCount} = cancerStudy;
                 const max = cancerStudySet.studies.length;
-                const colors = ['1E36BF', '128C47', 'BF2231', '7D1FBF', 'BF7D15'];
-                const color = this.lightenDarkenColor(colors[i%5], (j + 1)/max * 120);
+                if (cancerStudySet.shortName === "Prostate") int = 18;
+                if (cancerStudySet.shortName === "Ovary" || cancerStudySet.shortName == "Uterus" || cancerStudySet.shortName == "Stomach") int = 0;
```

Comment:
@Tamba32 Why are these hardcoded integers necessary? Can we get around relying on specific `shortName` of studies? Looks like it this code is mainly for particular colors, not the `shortName` of a study? Ideally we would come up with a more general solution, not hardcoding any color or study cases

`int` is a bit of an odd name for a variable